src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx

```diff
@@ -30,6 +30,19 @@ export const RoomModalOngoingEvent: React.FC<RoomModalOngoingEventProps> = ({
     roomEvents.length > 0 &&
     (currentEvent ? currentEvent : roomEvents[0]);
 
+  const joinRoom = useCallback(() => {
+    enterRoom();
+    openUrl(getRoomUrl(room.url));
+  }, [enterRoom, room.url]);
+
+  const triggerAttendance = useCallback(() => {
+    dispatch(retainAttendance(true));
+  }, [dispatch]);
+
+  const clearAttendance = useCallback(() => {
+    dispatch(retainAttendance(false));
+  }, [dispatch]);
```

Comment:
DRY with `src/components/templates/Camp/components/RoomModalOngoingEvent.tsx`, but since I believe that is one that we will be removing shortly, this is fine.